decidim-proposals/spec/shared/manage_proposals_examples.rb

```diff
@@ -29,13 +29,12 @@
   context "creation" do
     context "when official_proposals setting is enabled" do
       before do
-        current_feature.settings[:official_proposals_enabled] = true
-        current_feature.save
+        current_feature.update_attributes!(settings: { official_proposals_enabled: true })
```

Comment:
Well, I actually hadn't noticed that... Good point. Although, as you point out, being a spec that's probably what we want here.